subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -208,24 +208,35 @@ if NRF_CLOUD_AGPS
 config NRF_CLOUD_AGPS_AUTO
 	bool "Automatically request A-GPS on bootup"
 
-config NRF_CLOUD_AGPS_SINGLE_CELL_ONLY
-	bool "Obtain approximate location using a single cell tower instead of GPS"
+module = NRF_CLOUD_AGPS
+module-str = nRF Cloud A-GPS
+source "subsys/logging/Kconfig.template.log_config"
+
+endif # NRF_CLOUD_AGPS
+
+endmenu
+
+menu "nRF Cloud Cellular Positioning"
 
-config NRF_CLOUD_AGPS_REQ_CELL_BASED_LOC
+config NRF_CLOUD_CELL_POS
+	bool "Enable nRF Cloud Cellular Positioning"
+	imply FPU
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_NETWORK
+
+if NRF_CLOUD_CELL_POS
+
+config NRF_CLOUD_CELL_POS_REQ_CELL_BASED_LOC
```

Comment:
suggest renaming to just `NRF_CLOUD_CELL_POS_REQ_LOC`

Reply:
I suggest to remove this CONFIG as the decision is given in nrf_cloud_request_cell_location() by APP instead